### ISSUE TITLE
rename/alias eqv to equivalent

### ISSUE THREAD

**Issue by maiermic** [traitsjs/traits.js#2]
All methods on the `Trait` interface are self-explaining thanks to full words instead of abbreviations. The only exception is `eqv`. I suggest to rename `eqv` to `equivalent`. Thus, the interface would be consistent. You can keep `eqv` as deprecated alias to be backward compatible.

**Comment by dotnetCarpenter** [comment]
@maiermic I don't think that `eqv` should be deprecated but making `equivalent` an alias for `eqv` is a good idea and could be in a minor release.

**Comment by maiermic** [comment]
@dotnetCarpenter My intention is to point out to users of *traits.js* to prefer `equivalent` to `eqv`. If I'm not mistaken, deprecating is a common practice even for method naming:

> Since merely renaming an item would break backwards compatibility, the existing name must be left in place. The original name will likely remain indefinitely, but will be deprecated to encourage use of the newer, more consistent naming convention.
[Software deprecation - Wikipedia](https://en.wikipedia.org/wiki/Deprecation#Software_deprecation)

**Comment by dotnetCarpenter** [comment]
@maiermic If you want to write `equivalent` to achieve better readability, I'm all for it. But I for one will not use it, since English is not my first language and `equivalent` doesn't give me anything oppose to `eqv`, which I find easier to spell and remember. So I won't deprecate `eqv`.

What I'm trying to say is that I will add the alias but it is very low on my priority list. If you want to add it, you're welcome to create a PR. If not, then I will add it eventually.
